crates/openapi/src/routes/payments.rs

```diff
@@ -846,3 +847,21 @@ pub(crate) enum ForceSync {
     security(("publishable_key" = []))
 )]
 pub fn list_payment_methods() {}
+
+/// Payments - List
+///
+/// To list the *payments*
+#[cfg(feature = "v2")]
+#[utoipa::path(
+    get,
+    path = "/v2/payments/list",
+    params(api_models::payments::PaymentListConstraints),
+    responses(
+        (status = 200, description = "Successfully retrieved a payment list", body = Vec<PaymentListResponse>),
```

Comment:
This is not a `Vec<PaymentListResponse>` right? It should have been `PaymentListResponse` instead?